Monika After Story/game/zz_selector.rpy

```diff
@@ -3346,8 +3469,14 @@ label mas_selector_sidebar_select(items, select_type, preview_selections=True, o
 
         # setup prev line
         prev_line = ""
+    
+        # setup filtering
+        item_type = None
+        show_filter = False
```

Comment:
I would rather we not have global variables affecting the screen. If you need something to maintain state within screen iterations, then add additional props to the mailbox that is used here.

Reply:
Fixed in new commit.